### ISSUE TITLE
Unexpected inner loop exception: missing keyword: has_browser_handler:

### ISSUE THREAD

**Issue by yjukaku** [twalpole/apparition#56]
Hello! Thanks for your work on this gem.

I'm working on getting my project running on CircleCI and I'm running into this exception. I saw this closed issue (#18) for a similar exception message, but in my case I believe it has to do with a dialog because the trace points to handling the `Page.javascriptDialogOpening` event.

There seems to be a required keyword argument here, `has_browser_handler:` which I'd guess is not being passed to Ruby (?):

```
@session.on 'Page.javascriptDialogOpening' do |type:, message:, has_browser_handler:, **params|
```

```
/home/circleci/project/vendor/bundle/ruby/2.4.0/gems/apparition-0.5.0/lib/capybara/apparition/page.rb:428:in `block in register_event_handlers'
/home/circleci/project/vendor/bundle/ruby/2.4.0/gems/apparition-0.5.0/lib/capybara/apparition/driver/chrome_client.rb:211:in `block in process_handlers'
/home/circleci/project/vendor/bundle/ruby/2.4.0/gems/apparition-0.5.0/lib/capybara/apparition/driver/chrome_client.rb:207:in `each'
/home/circleci/project/vendor/bundle/ruby/2.4.0/gems/apparition-0.5.0/lib/capybara/apparition/driver/chrome_client.rb:207:in `process_handlers'
/home/circleci/project/vendor/bundle/ruby/2.4.0/gems/apparition-0.5.0/lib/capybara/apparition/driver/chrome_client.rb:185:in `block in process_messages'
/home/circleci/project/vendor/bundle/ruby/2.4.0/gems/apparition-0.5.0/lib/capybara/apparition/driver/chrome_client.rb:175:in `loop'
/home/circleci/project/vendor/bundle/ruby/2.4.0/gems/apparition-0.5.0/lib/capybara/apparition/driver/chrome_client.rb:175:in `process_messages'
/home/circleci/project/vendor/bundle/ruby/2.4.0/gems/apparition-0.5.0/lib/capybara/apparition/driver/chrome_client.rb:217:in `block in start_threads'
```

Any suggestions?

**Comment by yjukaku** [comment]
The method I'm calling which seems to trigger this exception is `accept_confirm` [defined by Capybara](https://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSession:accept_confirm).

This spec passes locally, but not on CircleCI, which is a little strange. I'm running with `headless: true` locally as well.

**Comment by twalpole** [comment]
It sounds like the versions of Chrome being tested against are probably different, or the CDP specification has changed

**Comment by twalpole** [comment]
@yjukaku Note - the docs for CDP show that hasBrowserHandler will be passed - https://chromedevtools.github.io/devtools-protocol/tot/Page/#event-javascriptDialogOpening

**Comment by yjukaku** [comment]
That's strange then that it's failing on CircleCI. Perhaps the option was added to CDP at a later point. Here's the image I'm using on CircleCI: https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/ruby/images/2.4.3-stretch/node-browsers/Dockerfile

I'm a newbie to Docker, but it looks to me that it installs a specific version of Chrome ([chrome-stable_64.0.3282.186-1_amd64](https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_64.0.3282.186-1_amd64.deb) but the latest version of`chromedriver`... so that's probably the source of the issue, no? Chromedriver must have added that parameter, but the older version of Chrome is not passing it?

In any case, would you accept a PR that defaults the `has_browser_handler` keyword argument to `false`?

**Comment by twalpole** [comment]
@yiukaku Chrome 64 was released 2.5 years ago - Since chrome is an evergreen browser we aren't supporting versions that old. Therefore, no I won't be accepting PR that defaults `has_browser_handler` because there's tons of other stuff in CDP that won't work with a version of Chrome that old either.
